### PR TITLE
Plugins, cURL, fixes

### PR DESCRIPTION
**Pull request by ConsM**

### CODE REVIEW

File: tyk-docs/content/customise-tyk/plugins/index.md [hunk 1]
```diff
@@ -26,7 +26,15 @@ All plugins require the following addition to be made to your `tyk.conf` file:
...
-For a gRPC rich plugin a further parameter is required within coprocess_options:
+`enable_coprocess` enables the rich plugins feature.
+
+`enable_bundle_downloader` enables the bundle downloader.
+
+`bundle_base_url` is a base URL that will be used to download the bundle.
+
+`public_key_path` sets a public key, this is used for verifying signed bundles. You may omit this if unsigned bundles are used.
+
+For a [gRPC][4] rich plugin a further `coprocess_grpc_server` parameter is required within coprocess_options:
```
Comment (ConsM) [inline tyk-docs/content/customise-tyk/plugins/index.md]: put coprocess_options in a code block

File: tyk-docs/content/customise-tyk/plugins/javascript-middleware/install-middleware/index.md [hunk 2]
```diff
@@ -10,3 +10,6 @@ url: "/customise-tyk/plugins/javascript-middleware/install-middleware"
 
 Installing middleware is different for different releases of Tyk, for example, in Tyk Community Edition it is possible to directly specify a path to a file in the API Definition, while in Tyk Pro, we recommend using a directory-based loader.
 
+> **NOTE:** Tyk Cloud currently does not support custom middleware loading. However, it is possible with a [Tyk Hybrid deployment][1].
+
+[1]: /customise-tyk/javascript-middleware/install-middleware/tyk-hybrid/
```
Comment (ConsM) [inline tyk-docs/content/customise-tyk/plugins/javascript-middleware/install-middleware/index.md]: this link should be /docs/customise-tyk/plugins/javascript-middleware/install-middleware/tyk-hybrid/

File: tyk-docs/content/customise-tyk/plugins/rich-plugins/python/tutorial-add-demo-plugin-api.md [hunk 3]
```diff
@@ -57,15 +57,19 @@ To enable Python plugins you need to add the following block to `tyk.conf`:
...
-    "bundle_base_url": "http://my-bundle-server.com/bundles/",
+    "bundle_base_url": "http://dummy-bundle-server.com/bundles/",
     "public_key_path": "/path/to/my/pubkey",
 ```
 
 `enable_coprocess`: enables the rich plugins feature.
 
 `enable_bundle_downloader`: enables the bundle downloader.
 
+<<<<<<< HEAD
```
Comment (ConsM) [inline tyk-docs/content/customise-tyk/plugins/rich-plugins/python/tutorial-add-demo-plugin-api.md]: missed one conflict here

File: tyk-docs/content/manage-multiple-environments/move-policies-between-environments.md [hunk 4]
```diff
@@ -16,8 +16,8 @@ First you must set up your new environment to respect explicit policy IDs. To do
 ### Step 1: Get your Policy
 
 ```
-    curl -X GET -H "authorization: {YOUR TOKEN}"
-     -s
+    curl -X GET -H "authorization: {YOUR TOKEN}" \
+     -s \
      -H "Content-Type: application/json"
```
Comment (ConsM) [inline tyk-docs/content/manage-multiple-environments/move-policies-between-environments.md]: a backslash at the end of this line

File: tyk-docs/content/manage-multiple-environments/move-policies-between-environments.md [hunk 5]
```diff
@@ -102,9 +102,9 @@ Remove the `_id` field and put the value of the `_id` field into the `id` field,
 Save the new `policies.json` file and then let's POST it back to the new environment:
 
 ```
-    curl -X POST -H "authorization: {API-TOKEN}"
-     -s
-     -H "Content-Type: application/json"
+    curl -X POST -H "authorization: {API-TOKEN}" \
+     -s \
+     -H "Content-Type: application/json" \
      -d @policies.json
```
Comment (ConsM) [inline tyk-docs/content/manage-multiple-environments/move-policies-between-environments.md]: a backslash at the end of this line

File: tyk-docs/content/tyk-dashboard-api/oauth-key-management.md [hunk 6]
```diff
@@ -97,14 +99,16 @@ Any OAuth keys must be generated under an API in the Dashboard. Any POST request
 | Body         | NONE                                       |
 
 ```
-    curl -vX GET -H "Authorization: {{API Access Credentials}}" -H "Content-Type: application/json" 
+    curl -vX GET -H "Authorization: {{API Access Credentials}}" \
+      -H "Content-Type: application/json"
```
Comment (ConsM) [inline tyk-docs/content/tyk-dashboard-api/oauth-key-management.md]: backslash at the end of the line

File: tyk-docs/content/tyk-dashboard-api/oauth-key-management.md [hunk 7]
```diff
@@ -97,14 +99,16 @@ Any OAuth keys must be generated under an API in the Dashboard. Any POST request
...
+      -H "Content-Type: application/json" 
       http://localhost:3000/api/apis/oauth/{{api-id}}/{{client_id}}
 ```
 
 #### Sample Request
 
 ```
-    curl -vX GET -H "Authorization: 68525fff302641704e1a737bfca088da" -H "Content-Type: application/json" 
+    curl -vX GET -H "Authorization: 68525fff302641704e1a737bfca088da" \
+      -H "Content-Type: application/json"
```
Comment (ConsM) [inline tyk-docs/content/tyk-dashboard-api/oauth-key-management.md]: backslash at the end of the line

File: tyk-docs/content/tyk-dashboard-api/oauth-key-management.md [hunk 8]
```diff
@@ -131,14 +135,15 @@ You can delete an OAuth client using a simple DELETE method. Please note that to
 | Body         | NONE                                       |
 
 ```
-    curl -vX DELETE -H "Authorization: {{API Access Credentials}}" -H "Content-Type: application/json" 
+    curl -vX DELETE -H "Authorization: {{API Access Credentials}}" \
+      -H "Content-Type: application/json"
```
Comment (ConsM) [inline tyk-docs/content/tyk-dashboard-api/oauth-key-management.md]: backslash at the end of the line

File: tyk-docs/content/customise-tyk/plugins/index.md (hunk 1 above)
Comment (marksou) [reply tyk-docs/content/customise-tyk/plugins/index.md]: Is that necessary as there is a code sample above them?

Comment (ConsM) [reply tyk-docs/content/customise-tyk/plugins/index.md]: yes, it's a configuration option

**Pull request merged**